packages/storybook/src/nimble/select/select-opened-matrix.stories.ts

```diff
@@ -47,51 +45,71 @@ interface SelectMatrixStoryOptions {
     grouped?: GroupedState;
     optionsOutsideGroup?: OptionsOutsideGroupState;
     slottedLabel?: boolean;
+    manyOptions?: boolean;
 }
 
 // prettier-ignore
 const component = ({
-    positionState, filterMode, loadingVisible, placeholder, grouped, optionsOutsideGroup, slottedLabel
+    positionState, filterMode, loadingVisible, placeholder, grouped, optionsOutsideGroup, slottedLabel, manyOptions
 }: SelectMatrixStoryOptions): ViewTemplate => html`
     <${selectTag} open 
-        position="${() => positionState[0]}" 
-        style="${() => positionState[1]} width: 250px; ${isChromatic() ? '--ni-private-spinner-animation-play-state:paused' : ''}"
+        position="${() => positionState}"
+        style="${() => (positionState === DropdownPosition.below ? 'margin-bottom: 120px;' : `margin-top: ${manyOptions ? 430 : 180}px;`)}"
+        style="${isChromatic() ? '--ni-private-spinner-animation-play-state:paused' : ''}"
```

Comment:
In this screenshot it looks like animation has happened. Can you double check if this is working / needs to be applied elsewhere?

https://www.chromatic.com/test?appId=60e89457a987cf003efc0a5b&id=6670cfca62bf533e2c2727e6

![image](https://github.com/ni/nimble/assets/10500124/62fbb6b3-554c-4739-bf5b-5151d824eceb)

Reply:
Didn't find this mistake anywhere else.